pkg/agent/profile.go

```diff
@@ -185,7 +181,7 @@ func (p *CgroupProfiler) Run(ctx context.Context) error {
 		return fmt.Errorf("load bpf object: %w", err)
 	}
 
-	cgroup, err := os.Open(p.target.PerfEventCgroupPath())
+	cgroup, err := os.Open(string(p.target[model.LabelName("__cgroup_path__")]))
```

Comment:
This gives me a weird feeling. I would rather prefer the previous version.

Reply:
At the very least we should have used the constant. `const CgroupPathLabelName = model.LabelName("__cgroup_path__")`

I still think this should be a method and the implementation details should be hidden under the hood.

Reply:
Feel free to fix this, I agree that at the very least we should use the constant. We did intentionally move to this being a label-set so over all I do think this is more consistent, but I agree with the use of the constant.